README.md

```diff
@@ -14,13 +14,32 @@ TODO
 
 \subsubsection*{Cantor set}
 \subsubsection*{Cartesian space ${\mathbb R}^n$}
+The set of all $n$-tuples, $\{ (x_1, x_2, ... , x_n) : x_i \in \mathbb{R} \}$.\\
+The number $n$ is called the dimension of $\mathbb{R}^n$, and the $n$-tuples
+can be thought of as vectors, since they form a natural vector space.\\
```

Comment:
Do you mean "natural" in a technical sense here? I wouldn't use the word since it has a technical reading. 
I'd also just state that this space can be equipped with (the known) vector space structure, not "can be thought of"

Reply:
That's probably better.